tools/documentation-generation/generate-documentation.js

```diff
@@ -4,97 +4,203 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
-const fs = require("fs");
-const { Documentation, Language } = require("jsii-docgen");
-const path = require("path");
-
-const targetFolder = path.resolve(
-  __dirname,
-  "..",
-  "..",
-  "website",
-  "docs",
-  "cdktf",
-  "api-reference"
-);
-/**
- * If the documentation code has geerics they are denoted in the hand-written
- * documentation as <>, e.g. Record<string, string>. Some other documentation parts
- * use <thing> to signal something needs to be filled in here
- *
- * In a markdown context these get interpretet as an HTML tag, this is why we break them up here
- * <thing> becomes < thing > which is no longer an HTML tag.
- */
-function replaceAngleBracketsInDocumentation(docs) {
-  const lines = docs.split("\n");
-  const sanitizedLines = lines.map((doc) => {
-    const htmlTags = doc.split("<");
-
-    const sanitizedTags = htmlTags.map((tag) => {
-      if (
-        ["code", "a", "sup"].some(
-          (item) =>
-            (tag.startsWith(item) || tag.startsWith(`/${item}`)) &&
-            tag.includes(">")
-        )
-      ) {
-        return tag;
-      }
+import * as fs from "fs";
+import * as path from "path";
 
-      const fullTag = tag.substring(0, tag.indexOf(">"));
+import { Documentation, Language } from "jsii-docgen";
 
-      return tag.replace(fullTag, ` ${fullTag} `);
-    });
+(async function () {
+  const remarkParse = (await import("remark-parse")).default;
+  const remarkStringify = (await import("remark-stringify")).default;
+  const visit = (await import("unist-util-visit")).visit;
+  const unified = (await import("unified")).unified;
```

Comment:
Do we need to add these packages as dependencies in `tools/documentation-generation/package.json`?

Reply:
Yeah, I probably added them in the next commit or so on the original branch. I'll also add them here 👍